src/js/pyodide.ts

```diff
@@ -254,7 +299,7 @@ export async function loadPyodide(
   config.env.PYTHONINSPECT ??= "1";
   const emscriptenSettings = createSettings(config);
   const API = emscriptenSettings.API;
-  API.lockFilePromise = loadLockFile(config.lockFileURL);
+  API.lockFilePromise = Promise.resolve(options_.lockFileContents);
```

Comment:
It is a bit confusing to have all of `config`, `option`, and `option_` here. But I guess I'll be able clean it up in #5746.